metaflow/plugins/kfp/tests/Dockerfile

```diff
@@ -1,8 +1,15 @@
 FROM analytics-docker.artifactory.zgtools.net/artificial-intelligence/ai-platform/aip-py39-cpu:4.0.1380
```

Comment:
Add comment describing the dual purpose of this Dockerfile: as both a CICD image and on Kubeflow.